tests/posts.test.ts

```diff
@@ -1,9 +1,10 @@
-const request = require("supertest");
-const initApp = require("../server.js");
-const mongoose = require("mongoose");
-const postsModel = require("../models/posts_model");
+import request from 'supertest';
+import initApp from '../server';
+import mongoose from 'mongoose';
+import postsModel, { PostDocument } from '../src/models/posts_model';
+
+let app: any;  
```

Comment:
```suggestion
let app: Express;  
```

You can use `import {Express} from "express";`
Same for `comments.test.ts` and `users.test.ts`